locales/fr_FR/pcsx2_Main.po

```diff
@@ -1667,49 +1661,47 @@ msgstr "&Connexion en cours..."
 
 #: pcsx2/gui/MainFrame.cpp:556
 msgid "Create &Blockdump"
-msgstr ""
+msgstr "Créer un &Blockdump"
 
 #: pcsx2/gui/MainFrame.cpp:556
 msgid "Creates a block dump for debugging purposes."
-msgstr ""
+msgstr "Crée un block dump pour des raisons de débogage."
 
 #: pcsx2/gui/MainFrame.cpp:560
-#, fuzzy
 msgid "Video"
-msgstr "&Vidéo (GS)"
+msgstr "Vidéo"
 
 #: pcsx2/gui/MainFrame.cpp:561
 msgid "Start Recording"
-msgstr ""
+msgstr "Démarrer l'enregistrement"
 
 #: pcsx2/gui/MainFrame.cpp:562
 msgid "Stop Recording"
-msgstr ""
+msgstr "Terminer l'enregitrement"
 
 #: pcsx2/gui/MainFrame.cpp:564
-#, fuzzy
 msgid "Screenshot"
-msgstr "Screenshots :"
+msgstr "Captures d'écran"
 
 #: pcsx2/gui/MainFrame.cpp:569
 msgid "New"
-msgstr ""
+msgstr "Nouveau"
 
 #: pcsx2/gui/MainFrame.cpp:570
 msgid "Stop"
-msgstr ""
+msgstr "Stopper"
```

Comment:
Arrêter ou Cesser?
Nitpicky. Seems okay.

Reply:
Without having the emulator in front of us, and without really knowing where does "Stop" appears, it's difficult to make its own choice. I had supposed "Stopper" was the universal solution.